### PR TITLE
added a guide to optimize queries

### PR DESCRIPTION
**Pull request by SusmeetJain**

### CODE REVIEW

File: docs/api/quick-start/optimization.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: Optimizing Your Usage
+description: Optimize your usage with the Dune API!
+---
+
+Here are some simple tips to optimize your usage (and billing) with the Dune API.
+
+### The LIMIT keyword
+
+Until your code looks ready and has been tested to fetch the data you need, use the `LIMIT` keyword at the end of any SQL query you are fetching data from.
```
Comment (davidkell) [inline docs/api/quick-start/optimization.md:10]: Here you can provide the context - as the Dune API bills by datapoints

File: docs/api/quick-start/optimization.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+``` sql title="example query"
+SELECT * FROM nft.trades;
+```
+Here is how you can run the same query but only fetch the first ten rows.
+
+``` sql title="example query with LIMIT"
+SELECT * FROM nft.trades
+LIMIT 10;
+```
+If you are using queries prepared by accounts other than yours, you can fork and then edit those queries to add this little change.
```
Comment (davidkell) [inline docs/api/quick-start/optimization.md:23]: remove "little"

File: docs/api/quick-start/optimization.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+Here is the same example query from the previous section. But instead of hardcoding the `date` we are filtering with, we pass it as a parameter. You can learn more about parameterized queries in [this section](../getting-started/queries/parameters.md) of our docs.
+
+``` sql title="example query with parameter based filtering"
+SELECT * FROM nft.trades
+WHERE "block_time" > '{{date}}'
+```
+
+!!! Local Storage
+    If there is data that you frequently need to call with the API. You can store it locally. And use your local database for that data. You can also keep updating this data dynamically using the tips described in this section, particularly parameterized queries.
```
Comment (davidkell) [inline docs/api/quick-start/optimization.md:48]: Some of these full stops don't make sense

e.g. "If there is data that you frequently need to call with the API, we recommend that you can store it locally e.g. in a database."

**Review by davidkell — CHANGES_REQUESTED** [review]

**Comment by andrewhong5297** [comment]
this may need to be updated to the new routing paths before merging

**Pull request closed without merging**